tests/simple_online.py

```diff
@@ -41,5 +43,12 @@ def main(args):
     parser.add_argument(
         "--max-tokens-per-minute", type=int, help="Max tokens per minute", default=None
     )
+    parser.add_argument("--max-retries", type=int, help="Max retries", default=None)
+    parser.add_argument(
+        "--require-all-responses",
+        action="store_true",
```

Comment:
Sorry what is this `store_true`? Can you not use bool?

Reply:
This stores a boolean